domains/transfer/TransferAgreementAutomation/Worker.UnitTests/Service/ProjectOriginWalletServiceTest.cs

```diff
@@ -32,13 +32,18 @@ public ProjectOriginWalletServiceTest()
     [Fact]
     public async Task TransferCertificates_CertificateNotTransfer_ShouldSetMetric()
     {
-        var transferAgreement = new TransferAgreementDto(
-            EndDate: DateTimeOffset.UtcNow.AddHours(3).ToUnixTimeSeconds(),
-            ReceiverReference: Guid.NewGuid().ToString(),
-            ReceiverTin: "12345678",
-            SenderId: Guid.NewGuid().ToString(),
-            StartDate: DateTimeOffset.UtcNow.ToUnixTimeSeconds()
-        );
+        var transferAgreement = new TransferAgreement
+        {
+            EndDate = DateTimeOffset.UtcNow.AddHours(3),
+            ReceiverReference = Guid.NewGuid(),
+            ReceiverTin = "12345678",
+            SenderId = Guid.NewGuid(),
+            StartDate = DateTimeOffset.UtcNow,
+            Id = Guid.NewGuid(),
+            SenderName = "SomeSender",
```

Comment:
Rename SomeSender to Peter Producent. "11223344" is his TIN, in all other tests we use, so we should use his name as well.

Reply:
Does it matter?